.github/workflows/execute_tutorial_notebooks_in_external_repo.yml

```diff
@@ -0,0 +1,88 @@
+########################################################################################
+# Trigger the notebooks' execution in the external repository.
+#
+# If the "main" branch triggered this event, then the nb execution is triggered in the
+# "main" branch, otherwise in the dev branch.
+#
+# Naming convention used in this workflow:
+#
+# "main repository": Project repository with the python package and the documentation.
+# "linked repository": Repository where the example gallery of jupyter notebooks is
+#                      stored.
+########################################################################################
+name: Trigger notebook execution in the external repository
+
+env:
+  # URL for the external repository linked with the notebooks in this project.
+  LINKED_REPO: pysteps/pysteps_tutorials
+  THIS_REPO: pysteps/pysteps
+
+on:
+  push:
+    branches: [ main, dev ]
+  pull_request:
+    branches: [ main, dev ]
```

Comment:
any push in any branch will trigger or only pushes within pull requests?

Reply:
With that configuration only pushes, or PR, to the `main` or the `dev`. Other branches are disabled by default. In that way, we can control if other development branches should trigger the examples rendering or not.